src/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -190,7 +190,7 @@
       <Version>2.2.2</Version>
     </PackageReference>
     <PackageReference Include="NuGetGallery.Core">
-      <Version>4.4.4-master-41290</Version>
+      <Version>4.4.5-master-2699519</Version>
```

Comment:
This dependency comes into NuGet.Services.AzureSearch via NuGet.Services.Validation.Common.Job. Are the versions the same? If not, how do we know there are no runtime issues?

Reply:
Looked a bit more closely into this, turns out `NuGet.Indexing` only needs `NuGet.Services.Entities`. Replaced the dependency here, and [aligned](https://github.com/NuGet/NuGet.Services.Metadata/pull/531/files#diff-bec7f637013541c5349ba50fe16cb67aR262) the `NuGet.Services.Entities` reference in `NuGet.Services.Metadata.Catalog` accordingly